tcp_tests/fixtures/openstack_fixtures.py

```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.fixture(scope='function')
-def openstack_actions(config, underlay):
+def openstack_actions(config, underlay, salt_actions):
```

Comment:
It's better to use 'salt_deployed' fixture instead just 'salt_actions' to ensure that salt is actually deployed. Fixture 'salt_deployed' returns the same object as salt_actions.